js/samples/uwp/SampleApp/MainPage.xaml.cs

```diff
@@ -24,16 +24,53 @@ public sealed partial class MainPage : Page
     {
         public MainPage()
         {
+            options.Add(new KeyValuePair<string, bool>("disableGrammar", false));
+            options.Add(new KeyValuePair<string, bool>("disableTranslation", false));
+            options.Add(new KeyValuePair<string, bool>("disableLanguageDetection", false));
+
             this.InitializeComponent();
         }
 
+        private List<KeyValuePair<string, bool>> options = new List<KeyValuePair<string, bool>>();
+
         private async void Button_Click(object sender, RoutedEventArgs e)
         {
             immersiveReader.TenantId = TenantId.Text;
             immersiveReader.ClientId = ClientId.Text;
             immersiveReader.ClientSecret = ClientSecret.Password;
             immersiveReader.Subdomain = Subdomain.Text;
-            await immersiveReader.Start("Title", ReaderContent.Text);
+            await immersiveReader.Start("Title", ReaderContent.Text, options, Language.Text);
+        }
+
+        private void RadioButtonOnSelectionHandler(object sender, RoutedEventArgs e)
+        {
+            var currentOption = ((RadioButton)sender);
```

Comment:
make this internal loop a helper fcn, since it seems to be shared between the two handlers?

Reply:
correct, done